helm/gateway-api-config/templates/clusterissuer.yaml

```diff
@@ -13,4 +13,4 @@ spec:
     - http01:
         gatewayHTTPRoute:
           parentRefs:
-          - name: {{ .Values.gateway.name }}
+          - name: gateway-name
```

Comment:
changing this temporarily because .Value.gateway is an array now. I'll refactor the clusterissuer in another PR